plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAlluxioCacheFileOperations.java

```diff
@@ -234,6 +244,11 @@ public static CacheOperation create(String operationName, Attributes attributes)
                 }
             }
             else {
+                // Checkpoint file lengths are unstable
```

Comment:
I didn't get why this would be the case, aren't checkpoint files immutable ?

Reply:
When repeatedly running the test we get checkpoint files of potentially different lengths in the runs. I believe this is  because of different compressibility of the timestamps and paths in the checkpoint.

Reply:
Pls add a comment - without the comment it may be cryptical to maintain.